### DISCUSSION TITLE
Welcome to JSON-goLD Discussions!

### DISCUSSION THREAD

**Discussion by kazarena** [piprate/json-gold#60] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with the community of JSON-goLD users. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us how you use JSON-goLD to interact with JSON-LD data in Go.

**Comment by reinkrul** [comment]
Hi there! 👋 

At Nuts Foundation (https://github.com/nuts-foundation/) we use JSON-goLD to validate/process DID documents and Verifiable Credentials (https://www.w3.org/TR/vc-data-model/) to provide SSI on Nuts. Nuts is an open source initiative by Dutch EHR (medical records) vendors to create an open, decentralized network to exchange patient health records.

We both write specifications (which use JSON-LD through DID documents and Verifiable Credentials) and a reference implementation in Go. Both are open source.

**Reply by kazarena** [reply]
Welcome, @reinkrul!

We are @piprate looked at EHR+DID+Verifiable Credentials as a use case in the early days and found it very useful and interesting. Best of luck with your product!

Thank you for the PR. I'm planning to open a discussion about requirements for json-gold/v2 in the next couple of week. It would be great to hear your feedback, as an active user.
